src/main/java/de/dataelementhub/rest/controller/v1/ElementController.java

```diff
@@ -355,4 +355,30 @@ public ResponseEntity getMembers(
       return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
   }
+
+  /**
+   * Update dataElementGroup or record members.
+   * If changes acquired return the new location otherwise return the old one.
+   */
+  @PutMapping("/{urn}/members")
```

Comment:
Not sure if PUT is the correct method. Maybe a POST to /urn/updateMembers or something like that...or a PATCH? We should discuss this.